routes/application.router.js

```diff
@@ -0,0 +1,11 @@
+module.exports = context => {
+  const { success } = require('../utils/envelope');
+  const { Router, middleware } = context;
+  const { auth } = middleware;
+  const version = process.env.npm_package_version;
+
+  return new Router()
+    .get('/application/version', auth(), async ctx => { 
```

Comment:
Is this page available to non-logged in users?

Reply:
No it is not.